custom_model_runner/datarobot_drum/drum/artifact_predictors/sklearn_predictor.py

```diff
@@ -59,14 +60,9 @@ def predict(self, data, model, **kwargs):
 
         if self.target_type.value in TargetType.CLASSIFICATION.value:
             if hasattr(model, "classes_"):
-                if set(str(label) for label in model.classes_) != set(
-                    str(label) for label in self.class_labels
-                ):
-                    error_message = "Wrong class labels {}. Use class labels detected by sklearn model: {}".format(
-                        self.class_labels, model.classes_
-                    )
-                    raise DrumCommonException(error_message)
-                labels_to_use = model.classes_
+                labels_to_use = marshal_labels(
+                    expected_labels=list(self.class_labels), actual_labels=list(model.classes_)
+                )
```

Comment:
I don't think we need to change labels_to_use here. This only covers sklearn models. The check/fix should be downstream from this